tests/src/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -306,4 +306,50 @@ class NodeJsActionContainerTests extends FlatSpec with Matchers {
         filtered(out).trim shouldBe empty
         filtered(err).trim shouldBe empty
     }
+
+    it should "error when requiring a non-existent package" in {
+        val (out, err) = withNodeJsContainer { c =>
+            val code = """
+                | function main(args) {
+                |     require('wildlyuniquenameofanonexistentpackage');
```

Comment:
But for now, thats not an issue as we are controlling which packages are available - so it was a mistake on my side - looks good to me otherwise :)